randovania/data/json_data/prime3.txt

```diff
@@ -1087,45 +1166,65 @@ Asset id: 8374398449894515797
   > Dock to Hidden Court
       Trivial
 
+> Event - Falls of Fire Wall; Heals? False
+  * Event Falls of Fire Wall
+  > Dock to G.F.S. Theseus
```

Comment:
Connect the event to hidden court instead, because to  trip one you have to drop down anyways.